test/runtests.jl

```diff
@@ -57,6 +57,7 @@ include("data.jl")
 include("mrsp.jl")
 include("rop.jl")
 include("restoration_redispatch.jl")
+include("utilization.jl")
```

Comment:
We might consider where the other algorithms will go down the road.  Should we put them all in one file, like `rop_heuristics.jl`?  If they will be in separate files maybe they should have a common prefix, like `rop_utilization`, `rop_rad`, ...

Reply:
I believe they should be in separate files, some of the other heuristics have specific helper functions and it will be better to keep them isolated to their own heuristics file.  

Renamed file to more easily group the rop heuristics together.